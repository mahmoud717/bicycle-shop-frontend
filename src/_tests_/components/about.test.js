import React from 'react';

import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import About from '../../components/about';

const mockStore = configureMockStore();
const store = mockStore({});
const setup = () => shallow(
  <Provider store={store}>
    <About />
  </Provider>,
);

describe('About Component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });
  test('renders the title text of the About Component', () => {
    const page = component.find('.about');
    expect(page).toMatchSnapshot();
  });
});
