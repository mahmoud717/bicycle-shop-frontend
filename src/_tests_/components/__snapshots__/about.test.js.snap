// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component renders the title text of the About Component 1`] = `ShallowWrapper {}`;
